Code/Framework/AzCore/AzCore/Script/ScriptSystemComponent.cpp

```diff
@@ -44,18 +45,17 @@ using namespace AZ;
  *      If the script was loaded by a ScriptComponent, Load will be called once reload is complete.
  */
 
-namespace
+
+// Called when a module has already been loaded
+static int LuaRequireLoadedModule(lua_State* l)
 {
-    // Called when a module has already been loaded
-    static int LuaRequireLoadedModule(lua_State* l)
-    {
-        // Push value to top of stack
-        lua_pushvalue(l, lua_upvalueindex(1));
+    // Push value to top of stack
+    lua_pushvalue(l, lua_upvalueindex(1));
 
-        return 1;
-    }
+    return 1;
 }
 
```

Comment:
This is an interesting case, at first look, either `namespace` or the `static` is redundant for `LuaRequireLoadedModule`

I personally prefer anonymous namespaces instead of statics, but we have to remember that those files are also part of the `unity` build.
To be ultra-careful of symbol clashes ( not sure if it's needed here ), one could do:
```cpp
namespace LocalTU_ScriptSystemComponent {
  static int foo() {};
}
```

Reply:
Anonymous namespaces and statics are going to have the same issue with `unity` builds of having the same symbol name.

The following will both caused duplicate symbol issues in unity builds if both their cpp files were included in the same unity file
```
namespace
{
    int foo() {};
}
// ...
namespace
{
    int foo() {};
}
```

```
static int foo() {};
// ...
static int foo() {};
```